internal/v1/handler_blueprints.go

```diff
@@ -788,13 +918,43 @@ func (h *Handlers) FixupBlueprint(ctx echo.Context, id openapi_types.UUID) error
 	}
 	desc := common.FromPtr(blueprintRequest.Description)
 
-	err = h.server.db.UpdateBlueprint(ctx.Request().Context(), uuid.New(), blueprintEntry.Id, userID.OrgID(), blueprintRequest.Name, desc, body)
+	slog.InfoContext(ctx.Request().Context(), "starting buildServiceSnapshots during fixup",
+		"blueprint_id", blueprintEntry.Id,
+		"distribution", blueprintRequest.Distribution,
+		"has_openscap", blueprintRequest.Customizations.Openscap != nil)
```

Comment:
Consider lowering amount of debug messages to just one at the end, or two begin/end of the function. If you want to keep this one okay, but please not info level these are turned on in production.